Marlin/src/pins/mega/pins_GT2560_V41b.h

```diff
@@ -51,14 +51,14 @@
  *  X AXIS              Y AXIS               Z1 AXIS            Z0 AXIS
  *    ---                 ---                 ---                 ---
  *   | 1 | 5V            | 1 | 5V            | 1 | 5V            | 1 | 5V
- *   | 2 | 24 X_MIN      | 2 | 28 Y_MIN      | 2 | PE7 Z1_MIN    | 2 | 30 Z0_MIN
+ *   | 2 | 24 X_MIN      | 2 | 28 Y_MIN      | 2 | 80 Z1_MIN    | 2 | 30 Z0_MIN
  *   | 3 | GND           | 3 | GND           | 3 | GND           | 3 | GND
  *    ---                 ---                 ---                 ---
  *     J3                  J4                  J5                  J6
  *
 */
 
-//#define Z1_MIN                             PE7  // Number??
+#define Z2_STOP_PIN                           80  // PE7 - Extended mega2560 pin
```

Comment:
OK, I see from the board schematic that there is only a MAX for one of the Z axis.

Reply:
I went ahead and changed this to be conditional like the other pins, in case someone wants to override it in their Configuration files.

I also merged in the latest bugfix-2.1.x so it could be processed with the latest updates of formatpins.